src/stdatamodels/jwst/datamodels/wcs_ref_models.py

```diff
@@ -459,6 +460,83 @@ def to_fits(self):
         raise NotImplementedError("FITS format is not supported for this file.")
 
 
+class MiriWFSSSpecwcsModel(ReferenceFileModel):
+    """A model for a reference file of type "specwcs" for MIRI WFSS."""
+
+    schema_url = "http://stsci.edu/schemas/jwst_datamodel/specwcs_miri_wfss.schema"
+    reftype = "specwcs"
+
+    def __init__(
+        self,
+        init=None,
+        displ=None,
+        dispx=None,
+        dispy=None,
+        invdispl=None,
```

Comment:
Where (if anywhere) is `invdispl` used?

Reply:
see https://github.com/spacetelescope/jwst/blob/f5e42aa105aa41b16faa4a3015afa4b6989fb517/jwst/assign_wcs/niriss.py#L456 where `lmodel` gets assigned to be `invdispl` for the backward transform